### ISSUE TITLE
Error - Document sections can only be emitted to web targets

### ISSUE THREAD

**Issue by bryaan** [Urigo/meteor-angular2.0-socially#48]
Why am I getting this error?

```
=> Errors prevented startup:                  

   While processing files with urigo:angular2-meteor (for target os.osx.x86_64):
   /tools/isobuild/compiler-plugin.js:418:13: Document sections can only be emitted to web targets
   at [object Object]._.extend.addHtml (/tools/isobuild/compiler-plugin.js:418:13)
   at [object Object]._.extend.addHtml (/tools/isobuild/compiler-plugin.js:294:24)
   at HtmlCompiler.processFilesForTarget (compilers/html_compiler.js:40:24)
```

**Comment by barbatus** [comment]
Check  this comment https://github.com/Urigo/angular-meteor/issues/996#issuecomment-184324494

**Comment by bryaan** [comment]
@barbatus If I download an example and have to modify it in order to run, isn't that a bug that should be fixed?

**Comment by derekkite** [comment]
I got this error with an html file in /test. When I removed it the error went away. I thought test was ignored when building.

`While processing files with urigo:angular2-meteor (for target os.linux.x86_64):
   /tools/isobuild/compiler-plugin.js:418:13: Document sections can only be emitted to web targets
   at [object Object]._.extend.addHtml (/tools/isobuild/compiler-plugin.js:418:13)
   at [object Object]._.extend.addHtml (/tools/isobuild/compiler-plugin.js:294:24)
   at HtmlCompiler.processFilesForTarget (compilers/html_compiler.js:40:24)
`
